unit-tests/src/test/java/org/eclipse/collections/impl/list/mutable/AbstractListTestCase.java

```diff
@@ -925,10 +925,9 @@ public void getWithIndexOutOfBoundsException()
     @Test
     public void getWithArrayIndexOutOfBoundsException()
     {
-        Object item = new Object();
-
         try
         {
+            Object item = new Object();
```

Comment:
Here's another scope reduction that I don't agree with.

Reply:
I can suppress it.